### ISSUE TITLE
call library function error 364

### ISSUE THREAD

**Issue by wudsh** [mattn/vimtweak#2]
hello mattn, there is a problem when i try to call setalpha, and i follow the step in vimtweak.c and  recompile the vimtweak.c in my computer ,but still got the problem. Would you help me to figure out the problem, and my gvim version is 8.0

**Comment by mattn** [comment]
please try to rebulid dll

**Comment by wudsh** [comment]
I did rebuild the vimtweak dll, following the code int vimtweak.c
 cl /LD vimtweak.c user32.lib
 gcc -shared -o vimtweak.dll vimtweak.c

**Comment by mattn** [comment]
Still have issue?

**Comment by wudsh** [comment]
yes, but the dll i rebuild is much smaller than the vimtweak32 or vimtweak64

**Comment by mattn** [comment]
because I built it on mingw64 not msvc, maybe.

**Comment by wudsh** [comment]
ok, i will have a try

**Comment by wudsh** [comment]
finally, it worked, and i have a new question, how to setup alpha at gvim startup, i wrote the code  call libcallnr("vimtweak.dll", "SetAlpha", 200) in gvimrc, but it not work

**Comment by mattn** [comment]
Are you trying to change alpha when given focus? then you can try this. https://github.com/mattn/transparency-windows-vim

**Comment by wudsh** [comment]
ありがとう it work perfect
